assets/sunshine.conf

```diff
@@ -1,318 +1,30 @@
-# If no external IP address is given, Sunshine will attempt to automatically detect external ip-address

```

Comment:
can't push your personal conf file to the project

Reply:
yes, noticed now. reverting